partiql-parser/Cargo.toml

```diff
@@ -23,6 +23,9 @@ version = "0.0.0"
 pest = "~2.1.3"
 pest_derive = "~2.1.0"
 thiserror = "~1.0.24"
+num-traits = "~0.2.14"
+num-bigint = "~0.4.0"
+bigdecimal = "~0.2.0"
```

Comment:
`num-traits` is explicitly pulled in to get `from_str_radix` which is in the `Num` trait.